<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AYARLAR MENÜSÜ</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Ana Pencere Rengi */
QDialog {
    background-color: #0f0c29;
}

/* Çerçeveler (Frames) - Görünmez olsun ama düzeni tutsun */
QFrame {
    border: none;
}

/* Yazılar (Labels) */
QLabel {
    color: #00d2ff; /* Neon Mavi */
    font-weight: bold;
    font-size: 14px;
    font-family: &quot;Segoe UI&quot;;
}

/* Yazı Kutuları (Inputs) */
QLineEdit {
    background-color: #1a1a2e;
    color: white;
    border: 1px solid #4b0082; /* Mor Çerçeve */
    border-radius: 5px;
    padding: 5px;
    min-height: 25px; /* Kutu ezilmesin diye yükseklik */
}
QLineEdit:focus {
    border: 1px solid #ff00cc; /* Tıklayınca pembe olsun */
}

/* Kaydet Butonu */
QPushButton {
    background-color: #2e1a1a;
    color: #00ff00;
    border: 2px solid #00ff00;
    border-radius: 10px;
    font-weight: bold;
    font-size: 16px;
    padding: 10px;
    min-width: 150px;
}
QPushButton:hover {
    background-color: #00ff00;
    color: black;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Normal Saat Ücreti</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txt_normal_ucret"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Normal Üye Ücreti</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txt_uye_ucret"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>VIP Saat Ücret</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="txt_ucret_vip"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>VIP Üye Ücreti</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="txt_ucret_vip_uye"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Çay Fiyatı</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txt_fiyat_cay"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Kahve Fiyatı</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txt_fiyat_kahve"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Tost Fiyatı</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="txt_fiyat_tost"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Kola Fiyatı</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="txt_fiyat_kola"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ayar_kaydet">
       <property name="text">
        <string>Fiyatları Güncelle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
